src/System.Net.Requests/tests/FileWebRequestTest.cs

```diff
@@ -80,7 +80,7 @@ public void GetRequestResponseAfterAbort_Throws()
         [Fact]
         public void NotImplementedMembers_Throws()
         {
-            WebRequest request = WebRequest.Create("file://anything");
+            WebRequest request = WebRequest.Create("file:///anything");
```

Comment:
This changed. On Unix, Unix paths must start with a '/' on Windows the behavior is unchanged.
https://github.com/dotnet/corefx/pull/15925/commits/00a1a716487f680f0b18f2b0f73791c1657d6fd2#diff-06f80dbdb7963930a25fb8a935d08e0dR2193